src/flamenco/features/feature_map.json

```diff
@@ -130,12 +129,12 @@
   {"name":"check_syscall_outputs_do_not_overlap","pubkey": "3uRVPBpyEJRo1emLCrq38eLRFGcu6uKSpUXqGvU8T7SZ","hardcoded":1},
   {"name":"enable_bpf_loader_set_authority_checked_ix","pubkey": "5x3825XS7M2A3Ekbn5VGGkvFoAg5qrRWkTrY4bARP1GL"},
   {"name":"enable_alt_bn128_syscall","pubkey": "A16q37opZdQMCbe5qJ6xpBB9usykfv8jZaMkxvZQi4GJ"},
+  {"name":"simplify_alt_bn128_syscall_error_codes","pubkey": "JDn5q3GBeqzvUa7z67BbmVHVdE3EbUAjvFep3weR3jxX","hardcoded":1},
+  {"name":"enable_alt_bn128_compression_syscall","pubkey": "EJJewYSddEEtSZHiqugnvhQHiWyZKjkFDQASd7oKSagn"},
   {"name":"enable_program_redeployment_cooldown","pubkey": "J4HFT8usBxpcF63y46t1upYobJgChmKyZPm5uTBRg25Z","hardcoded":1},
   {"name":"commission_updates_only_allowed_in_first_half_of_epoch","pubkey": "noRuG2kzACwgaY7TVmLRnUNPLKNVQE1fb7X55YWBehp"},
   {"name":"enable_turbine_fanout_experiments","pubkey": "D31EFnLgdiysi84Woo3of4JMu7VmasUS3Z7j9HYXCeLY"},
   {"name":"disable_turbine_fanout_experiments","pubkey": "Gz1aLrbeQ4Q6PTSafCZcGWZXz91yVRi7ASFzFEr1U4sa"},
-  {"name":"drop_merkle_shreds","pubkey": "84zy5N23Q9vTZuLc9h1HWUtyM9yCFV2SCmyP9W9C3yHZ"},
-  {"name":"keep_merkle_shreds","pubkey": "HyNQzc7TMNmRhpVHXqDGjpsHzeQie82mDQXSF9hj7nAH"},
```

Comment:
Did we intend on deleting these?  Shouldn't these remain in but hardcoded?

Reply:
fixed